common/app/model/content/Atoms.scala

```diff
@@ -27,6 +27,7 @@ final case class MediaAtom(
   assets: Seq[MediaAsset],
   title: String,
   duration: Option[Long],
+  isoDuration: Option[String],
```

Comment:
I would actually make `isoDuration` a `def` and also move the logic in this class.